experimental/dds/tree2/src/class-tree/schemaTypes.ts

```diff
@@ -109,13 +112,25 @@ export interface TreeNodeSchemaClass<
  * @beta
  */
 export interface TreeNodeSchemaCore<
-	out Name extends string = string,
-	out Kind extends NodeKind = NodeKind,
-	out Specification = unknown,
+	out Name extends string,
+	out Kind extends NodeKind,
+	out ImplicitlyConstructable extends boolean,
 > {
 	readonly identifier: Name;
 	readonly kind: Kind;
-	readonly info: Specification;
+	readonly info: unknown;
+
+	/**
+	 * If true, when constructing insertable content,
+	 * instead of including an {@link Unhydrated} node,
+	 * the data that would have been passed to the constructor can be included instead.
```

Comment:
```suggestion
	 * When constructing insertable content,
	 * data that could be passed to the node's constructor can be used instead of an {@link Unhydrated} node
	 * iff implicitlyConstructable is true.
```